test/browser_tests/page_objects/page_about-director.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+function _getQAElement( selector, getAllFlag ) {
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/8555b964c486de0b03084f0e58e3f97f44f7727a